examples/src/bin/advanced_serial.rs

```diff
@@ -4,10 +4,10 @@
 //! of the configuration change the output signal.
 //!
 //! The following wiring is assumed:
-//! - TX => GPIO4
-//! - RX => GPIO5
+//! - TX => GPIO4 (GPIO21 - `esp32p4`)
```

Comment:
Does this have to be special cased for the P4? We don't define 1/3 for ESP32 either.

Reply:
Sorry, which 1/3 pins are we talking about? 😅

Reply:
ESP32 rx/tx is GPIO 1 and 3 in whichever order.

Reply:
I'm more likely to agree with this, at least because most likely the version of P4 I'm using is not the final version and pin layout can still change 1000 times 😄